src/redturtle/prenotazioni/content/prenotazione.py

```diff
@@ -186,6 +186,13 @@ class IPrenotazione(model.Schema):
         required=False, title=_("label_booking_staff_notes", "Staff notes")
     )
 
+    # Schema is defined in PrenotaizioneType ad an datagridfield, and here we save the data as an json
+    # in base of selected type
+    additional_fields = schema.TextLine(
```

Comment:
why not using a JSONfield like blocks one?

Reply:
done